atlasdb-config/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
     compile project(':leader-election-impl')
     compile project(':lock-impl')
     compile project(':timestamp-client')
+    compile project(':qos-service-impl')
```

Comment:
This is needed as a `AtlasDBQosClient` is created in TMs. We should perhaps have a factory here that does the creation.

Reply:
i think this is fine - we already have the other `impl` projects here as well